File: PeachPie Extension Libraries/Peachpie.Base.Ext/.sdk/Php/Threading/Tasks/TaskAlreadyStartedException.php
<?php

namespace Php\Threading\Tasks;

use Exception;
use Throwable;

class TaskAlreadyStartedException extends Exception
{
    /**
     * @param string|null $message
     * @param int|null $inner
     * @param Throwable|null $previous
     * @throws TaskAlreadyStartedException
     */
    public function __construct(string $message = null, int $inner = null, Throwable $previous = null)
    {
        if($message == null && $inner == null && $previous ) {
            throw new TaskAlreadyStartedException("Task already started.");
        }
    }
}

File: PeachPie Extension Libraries/Peachpie.Base.Ext/src/Threading/Tasks/TaskAlreadyStartedException.cs

using Pchp.Library.Spl;

namespace Php.Threading.Tasks;

public class TaskAlreadyStartedException : Exception
{
    public TaskAlreadyStartedException()
        : base("Task already started.") { }

    public TaskAlreadyStartedException(string message)
        : base(message) { }

    public TaskAlreadyStartedException(string message, long inner)
        : base(message, inner) { }
}